packages/integrations/lit/server.js

```diff
@@ -1,4 +1,6 @@
+// It must stay at the top
```

Comment:
Explaining why would be great for anyone unaware of this pattern
```suggestion
// Separate import from the rest so it doesn't get re-organized after other imports
```
something like that